src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -535,15 +557,33 @@ const StudentApplication = ({ userProfile, authentication }) => {
    * @function submitApplication
    */
   const submitApplication = async () => {
-    if (applicationDetails.Applicants.every((applicant) => isApplicantValid(applicant))) {
+    const showGenericSubmitError = () => {
+      setSnackbarText('Something went wrong while trying to submit the application.');
+      setSnackbarSeverity('error');
+      setSnackbarOpen(true);
+    };
```

Comment:
It would be a quick and easy update to convert this to a general function `createSnackbar(message, severity)` that can be used throughout the entire application.

Reply:
Addresses in #1096